### PR TITLE
Obc addresses

### PR DESCRIPTION
**Pull request by Taylorr82**
We realized there may be a need to make each satellite have their own CSP node. This is a proposal on how to implement that

### CODE REVIEW

**Review by tganley — DISMISSED** [review]

File: src/groundStation/system.py [hunk 1]
```diff
@@ -67,15 +67,12 @@ def __init__(self):
         }
 
         self.APP_DICT = {
```
Comment (Taylorr82) [inline src/groundStation/system.py:69]: I don't like the name APP_DICT for this use. can we change it?

**Review by jmmabanta — COMMENTED** [review]
From what I'm seeing, it seems as though the only thing that this changes is the prefix of a command (the server it is sent to). 

If this is the only thing, would the website need to be changed to add a "Select Satellite" feature or would it make more sense for operators to use the current command syntax to specify the satellite when entering commands [ie. **ex2**.time_management.get_time()]? The latter is how the website currently behaves.

**Comment by Taylorr82** [comment]
I suppose it depends how you plan on doing the website. For "console" access I think typing the satellite prefix is acceptable since you'd have to type it anyway for eps commands. Would help the operator understand exactly what they're doing too

**Comment by jmmabanta** [comment]
I guess I'll keep the current behaviour where the operator has to explicitly specify the server in the command, until a feature comes up where satellite specification isn't as explicit

**Comment by Taylorr82** [comment]
Is this good to push then?

File: src/groundStation/groundStation.py [hunk 2]
```diff
@@ -324,6 +333,11 @@ def getOptions(self):
             type=int,
             default='15000', # 15 seconds
             help='RDP connection timeout')
+        self.parser.add_argument(
+            '--satellite',
```
Comment (jmmabanta) [inline src/groundStation/groundStation.py:337]:
```suggestion

        self.parser.add_argument(
            '-s',
            '--satellite',
```
Just to conform with other cli arguments and it makes it quicker to specify as an arg (unless it conflicts with something I don't know of)

**Review by jmmabanta — COMMENTED** [review]
Other than a minor QOL change to the cli argument it all looks good to me!

**Pull request merged**